docs/cli/installation.md

```diff
@@ -2,11 +2,12 @@
 
 To install the CLI use one of the following methods:
 
-=== "Homebrew"
-    `brew install cdk8s`
-
 === "npm"
     `npm i -g cdk8s-cli`
 
 === "yarn"
     `yarn global add cdk8s-cli`
+
+=== "Homebrew"
+    `brew install cdk8s`
+Note: the latest version of the `cdk8s` cli available via Homebrew is `0.33.0`. Homebrew does not accept beta versions, so until there is a stable release please use `npm` or `yarn` to install the latest cli.
```

Comment:
```suggestion
    > Note: the latest version of the `cdk8s` cli available via Homebrew is `0.33.0` because Homebrew does not accept beta versions. Once a stable `1.0.0` version is released it will be available on Homebrew as well. In the meantime, please use `npm` or `yarn` to install the latest cli.
```